src/content/docs/it/install/auto.mdx

```diff
@@ -0,0 +1,136 @@
+:::
+
+## 1. Esegui lo strumento di setup
+
+Esegui il seguente comando nel tuo terminale per eseguire il nostro utilie strumento di installazione:
+
+<PackageManagerTabs>
+  <Fragment slot="npm">
+  ```shell
+  # crea un nuovo progetto con npm
+  npm create astro@latest
+  ```
+  </Fragment>
+  <Fragment slot="pnpm">
+  ```shell
+  # crea un nuovo progetto con pnpm
+  pnpm create astro@latest
+  ```
+  </Fragment>
+  <Fragment slot="yarn">
+  ```shell
+  # crea un nuovo progetto con yarn
+  yarn create astro
+  ```
+  </Fragment>
+</PackageManagerTabs>
+
+Puoi eseguire il comando `create astro` ovunque nel tuo computer, perciò non serve creare una nuova cartella vuota per il tuo progetto prima di cominciare. Se ancora non hai una cartella vuota per il tuo progetto, lo strumento ne creerà automaticamente una al posto tuo.
+
+Se tutto va bene, dovresti vedere il messaggio "Liftoff confirmed. Explore your project!" ("Decollo confermato. Esplora il tuo progetto!") seguito da alcuni suggerimenti per i prossimi passi. Entra nella nuova cartella del tuo progetto con il comando `cd` per iniziare ad usare Astro.
+
+Se hai saltato il comando `npm install` durante l'esecuzione dello strumento CLI, allora assicurati di installare le dependencies prima di continuare.
+
+## 2. Fai partire Astro ✨
+
+Astro arriva con un server di sviluppo integrato che ha tutto il necessario per il progresso del tuo progetto. Il comando `astro dev` farà partire il server di sviluppo locale così che tu possa vedere il tuo sito in azione per la primissima volta.
+
+Ogni template ufficiale arriva con uno script preconfigurato che eseguirà il comando `astro dev` al posto tuo. Usa il tuo package manager preferito per eseguire questo comando e lanciare il server di sviluppo locale di Astro.
+
+<PackageManagerTabs>
+  <Fragment slot="npm">
+  ```shell
+  npm run dev
+  ```
+  </Fragment>
+  <Fragment slot="pnpm">
+  ```shell
+  pnpm run dev
+  ```
+  </Fragment>
+  <Fragment slot="yarn">
+  ```shell
+  yarn run dev
+  ```
+  </Fragment>
+</PackageManagerTabs>
+
+
+Se tutto va bene, ora Astro dovrebbe servire il tuo progetto all'indirizzo [http://localhost:3000/](http://localhost:3000/)!
+
+Astro osserverà ogni cambiamento ai file nella cartella `src/`, perciò non dovrai far ripartire il server ad ogni modifica nel progetto durante lo sviluppo.
+
+Se non riesci ad aprire il tuo progetto nel browser, torna al terminale dove hai eseguito il comando `dev` e controlla se c'è stato un errore, o se il tuo progetto viene servito ad un indirizzo diverso da quello indicato qua sopra.
+
+## Template di partenza
+
+Puoi anche iniziare un nuovo progetto di Astro partendo da un [esempio ufficiale](https://github.com/withastro/astro/tree/main/examples) o dal ramo `main` di una qualunque repository di GitHub passando l'argomento `--template` al comando `create astro`.
```

Comment:
Personally I'm used to saying "branch" but I've seen the literal translation in a few places like html.it, and some of my uni professors used it. Maybe because it's easier to understand for people that have had a lower level of English education.

We could keep the English word since most CS-related applications and tools use it. And Git itself doesn't translate the word (does it even have i18n?).